File: Code Generation/responses/python/deepseek/easy/unique-3-digit-even-numbers.py
from typing import List
from itertools import permutations

class Solution:
    def totalNumbers(self, digits: List[int]) -> int:
        unique_numbers = set()
        
        for perm in permutations(digits, 3):
            if perm[0] != 0 and perm[2] % 2 == 0:
                num = perm[0] * 100 + perm[1] * 10 + perm[2]
                unique_numbers.add(num)
        
        return len(unique_numbers)

File: Code Generation/responses/cpp/deepseek/easy/unique-3-digit-even-numbers.cpp
#include <vector>
#include <unordered_set>
#include <algorithm>

using namespace std;

class Solution {
public:
    int totalNumbers(vector<int> digits) {
        unordered_set<int> resultSet;
        int n = digits.size();
        
        for (int i = 0; i < n; ++i) {
            for (int j = 0; j < n; ++j) {
                for (int k = 0; k < n; ++k) {
                    if (i != j && i != k && j != k) {
                        int num = digits[i] * 100 + digits[j] * 10 + digits[k];
                        if (num >= 100 && num % 2 == 0) {
                            resultSet.insert(num);
                        }
                    }
                }
            }
        }
        
        return resultSet.size();
    }
};